auslib/db.py

```diff
@@ -1764,6 +1765,56 @@ def _putWatchdogValue(self, changed_by, value, where=None, transaction=None):
             super(Dockerflow, self).update(where=where, what=value, changed_by=changed_by, transaction=transaction)
 
 
+class Shutoffs(AUSTable):
+    def __init__(self, db, metadata, dialect):
+        self.table = Table('shutoffs', metadata,
+                           Column('shutoff_id', Integer, primary_key=True,
+                                  autoincrement=True),
+                           Column('product', String(15), nullable=False),
+                           Column('channel', String(75)),
+                           Column('mapping', String(100)),
+                           UniqueConstraint('product', 'channel')
+                           )
+        AUSTable.__init__(self, db, dialect, history=False, versioned=False)
```

Comment:
Did we talk at all about whether or not history should be enabled for this table? My gut says it should...

I think leaving versioning off is probably OK, since operations are idempotent.